src/main/webapp/view.js

```diff
@@ -14,7 +14,6 @@
 
 const PARAM_ID = 'id';
 
```

Comment:
Would it be better if these spaces are removed?

Reply:
Removed